tests/components/axis/test_device.py

```diff
@@ -385,31 +384,33 @@ async def test_shutdown():
 
     axis_device = axis.device.AxisNetworkDevice(hass, entry)
     axis_device.api = Mock()
+    axis_device.api.vapix.close = AsyncMock()
 
-    axis_device.shutdown(None)
+    await axis_device.shutdown(None)
 
     assert len(axis_device.api.stream.stop.mock_calls) == 1
+    assert len(axis_device.api.vapix.close.mock_calls) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)